xsd/datacite-kernel-4/metadata.xsd

```diff
@@ -524,7 +528,7 @@ Use the complete title of a license and include version information if applicabl
                     </xs:element>
                     <xs:element name="number" minOccurs="0">
                       <xs:annotation>
-                        <xs:documentation>Issue number or name of the related item.</xs:documentation>
+                        <xs:documentation>Number of the related item e.g. report number of article number.</xs:documentation>
```

Comment:
I think maybe this should be `e.g. report number or article number` (`or` instead of `of`).

Reply:
This file was downloaded from DataCite. We probably want to keep the statement as is.